src/lsm/manifest_level.zig

```diff
@@ -82,43 +99,62 @@ pub fn ManifestLevelType(
         }
 
         /// Set snapshot_max for the given table in the ManifestLevel.
-        ///
+        /// The table provided could either be a pointer to the table (from_level) in the
+        /// ManifestLevel or a pointer to a copy of that table (copy).
```

Comment:
```suggestion
        ///
```

This is implied by the type signature, no need to explain again.